## Deafult cost on states.
## Will be overwritten at runtime by first element in state cost list.
Q_pos_xy: 1
Q_pos_z: 1
Q_attitude: 1
Q_velocity: 1
Q_horizon: 1
Q_perception: 0

# Cost on inputs
R_thrust: 1
R_pitchroll: 1
R_yaw: 1

# Exponential scaling: W_i = W * exp(-i/N * cost_scaling).
# cost_scaling = 0 means no scaling W_i = W. 
state_cost_exponential: 0.0     # Scaling for state costs
input_cost_exponential: 0.0     # scaling for input costs

# Limits for inputs
max_bodyrate_xy: 10.0
max_bodyrate_z: 3.0
min_thrust: 0.1
max_thrust: 20.0

# Extrinsics for Perception Aware MPC
p_B_C: [ 0.0, 0.0, 0.0 ]               # camera in body center [m]
q_B_C: [ 0.6427876, 0, 0.7660444, 0 ]  # 45Â° pitched down

# Print information such as timing to terminal
print_info: false

# Bounds on state costs
B_pos_xy:   [   1.0,  1000.0  ]
B_pos_z:    [ 100.0,  1500.0  ]
B_attitude: [   1.0,   100.0  ]
B_velocity: [   1.0,  1000.0  ]
B_horizon:  [   1.0,     7.0  ]

### Circle ###
#L_pos_xy:     [   54.043  ]
#L_pos_z:      [  638.678  ]
#L_attitude:   [    3.684  ]
#L_velocity:   [    3.565  ]
#L_horizon:    [    4.00   ]

### Drop ###
#L_pos_xy:     [  174.28,     389.56,    155.74,     161.35  ]
#L_pos_z:      [  939.56,     715.74,    952.97,     906.38  ]
#L_attitude:   [    7.33,       8.89,     39.98,      24.98  ]
#L_velocity:   [   46.15,      25.77,     76.04,      22.76  ]
#L_horizon:    [    3.00,       4.00,      6.00,       5.00  ]

### Flip ###
#L_pos_xy:     [  165.68,     206.37,     382.77,    114.51  ]
#L_pos_z:      [  933.58,     887.64,     732.94,    994.08  ]
#L_attitude:   [    6.79,      13.84,      27.38,     12.64  ]
#L_velocity:   [   11.64,      16.68,      34.31,     11.50  ]
#L_horizon:    [    5.00,       2.00,       3.00,      1.00  ]

### Spiral ###
#L_pos_xy:     [  100.09,     58.15,      58.09  ]
#L_pos_z:      [  941.58,    692.82,     691.21  ]
#L_attitude:   [   55.16,     30.32,      34.36  ]
#L_velocity:   [   45.61,     11.26,       4.43  ]
#L_horizon:    [    4.00,      2.00,       2.00  ]

